### PR TITLE
 Tower Of Hanoi in TypeScript  

### LINKED ISSUE #2479: Tower of Hanoi
- [x] C
- [x] C++
- [x] C# - @biekymahato
- [ ] CoffeeScript
- [x] Dart - @disha2sinha
- [x] Go - @rbjoshi1309
- [x] Java
- [x] JS
- [x] Kotlin - @chandbud5
- [x] Php - @chandrikadeb7
- [x] Python
- [x] Readme
- [x] Ruby - @HardevKhandhar
- [ ] Rust
- [x] TypeScript - @vyom153069

### PR DESCRIPTION
**Pull request by vyom153069**
Fixes #2479

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Tower_Of_Hanoi.ts
- 

#### Languages Used:
- TypeScript
- 

#### Files Added:
- Comments added
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by vyom153069** [comment]
Please review my PR @jainaman224

File: Tower_Of_Hanoi/Tower_Of_Hanoi.ts [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+[LOG]: 55: Move Disk from Left to Middle. 
+[LOG]: 56: Move Disk from Left to Right. 
+[LOG]: 57: Move Disk from Middle to Right. 
+[LOG]: 58: Move Disk from Middle to Left. 
+[LOG]: 59: Move Disk from Right to Left. 
+[LOG]: 60: Move Disk from Middle to Right. 
+[LOG]: 61: Move Disk from Left to Middle. 
+[LOG]: 62: Move Disk from Left to Right. 
+[LOG]: 63: Move Disk from Middle to Right. 
+[ERR]: Executed JavaScript Failed:
```
Comment (raghav-dalmia) [inline Tower_Of_Hanoi/Tower_Of_Hanoi.ts:216]: remove these errors from the output section

File: Tower_Of_Hanoi/Tower_Of_Hanoi.ts [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+        solveHanoi(count - 1, from, other, to, move)
+        move(from, to)
+        //shift plat "other" to "from" using "to" plate
+        solveHanoi(count - 1, other, to, from, move)
+    }
+}
+/**/
+// Example: six discs
+var moveCount = 0
+solveHanoi(6, "Left", "Right", "Middle", (from , to) => {
```
Comment (raghav-dalmia) [inline Tower_Of_Hanoi/Tower_Of_Hanoi.ts:20]: run it for small value like 3 or 4

**Comment by raghav-dalmia** [comment]
make sure to squash your commits

**Comment by vyom153069** [comment]
@raghav-dalmia  i have a doubt , should i clone this repo or go for rebase(squashing the commits ) ?? or can i make another PR ?(because i have not enough internet to clone this repo)

**Comment by raghav-dalmia** [comment]
for `rebase` command you don't have to clone the repo again. You can also create new PR.

**Comment by vyom153069** [comment]
> 
> 
> for `rebase` command you don't have to clone the repo again. You can also create new PR.

i created a new PR please review it

**Comment by somya-kapoor** [comment]
Before creating a new PR, please close the exciting PR.

**Pull request closed without merging**